lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -1948,7 +1948,7 @@ enum class OptionalAdjustmentPosition : unsigned {
   /// Both return and parameter adjustments.
   ParamAndReturn = 4,
 };
-} // namespace
+} // end anonymous namespace.
```

Comment:
…and no period (for whatever reason). You can see it elsewhere in the codebase.

Reply:
Okay done!